src/server/repository/ahb.ts

```diff
@@ -31,17 +30,30 @@ export default class AHBRepository extends BlobStorageBacked {
     pruefi: string,
     formatVersion: string,
     type: FileType,
-  ): Promise<Ahb> {
+  ): Promise<Buffer> {
```

Comment:
Wieso wurde AHB in Buffer umbenannt?

Reply:
das AHB Objekt wurde an anderer Stelle schon definiert und returned den .json content. Buffer kann quasi auch binary sein und hier quasi (json | xlsx | csv).

Was man noch machen kann ist quasi beides abfragen `Promise<Ahb | Buffer>`

Reply:
Ist für mich erstmal fein.
Können wir später gerne optimieren